src/v1/text_to_speech_client.ts

```diff
@@ -11,22 +11,57 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+//
+// ** This file is automatically generated by gapic-generator-typescript. **
+// ** https://github.com/googleapis/gapic-generator-typescript **
+// ** All changes to this file may be overwritten. **
+
+import * as gax from 'google-gax';
+import * as path from 'path';
+
+import * as protosTypes from '../../protos/protos';
+import * as gapicConfig from './text_to_speech_client_config.json';
+
+const version = require('../../../package.json').version;
 
-'use strict';
+export interface ClientOptions
```

Comment:
Huh.  I would expect this interface to be kinda the same across all libraries.  Does this belong in gax?

Reply:
It does! Will move it there.

Reply:
will do. thanks! same for the interfaces below.

Reply:
moved common interfaces to gax and import them from micro-gen
ref: https://github.com/googleapis/gax-nodejs/commit/b93f9c106faecce337ef22c5864a9f2d45c80775